<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CClusterEditorDialog</class>
 <widget class="QDialog" name="CClusterEditorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Group editor</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>380</y>
     <width>181</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditClusterName">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>15</y>
     <width>191</width>
     <height>27</height>
    </rect>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>261</width>
     <height>321</height>
    </rect>
   </property>
   <property name="title">
    <string>Tests</string>
   </property>
   <widget class="QListView" name="listViewTests">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>211</width>
      <height>291</height>
     </rect>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::MultiSelection</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonAddTests">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>30</y>
      <width>24</width>
      <height>25</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Add test cases.</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
    <property name="arrowType">
     <enum>Qt::NoArrow</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonRemoveTests">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>60</y>
      <width>24</width>
      <height>25</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Remove test cases.</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>50</y>
     <width>261</width>
     <height>321</height>
    </rect>
   </property>
   <property name="title">
    <string>Code elements</string>
   </property>
   <widget class="QListView" name="listViewCEs">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>211</width>
      <height>291</height>
     </rect>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::MultiSelection</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonAddCEs">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>30</y>
      <width>24</width>
      <height>25</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Add code elements.</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonRemoveCEs">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>60</y>
      <width>24</width>
      <height>25</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Remove code elements.</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CClusterEditorDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CClusterEditorDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
